src/DependencyInjection/SonataMediaExtension.php

```diff
@@ -455,12 +455,18 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
             ;
 
             if (3 === $config['filesystem']['s3']['sdk_version']) {
-                $container->getDefinition('sonata.media.adapter.service.s3')
-                ->replaceArgument(0, [
-                    'credentials' => [
+                if (isset($config['filesystem']['s3']['secretKey']) && isset($config['filesystem']['s3']['secretKey'])) {
```

Comment:
you have the same isset 2 times?

Reply:
my bad, I test `secretKey`and `accessKey`